docs/plugins/create-a-plugin.md

```diff
@@ -39,3 +39,11 @@ yarn workspace @backstage/plugin-my-plugin start # Also supports --check
 This method of serving the plugin provides quicker iteration speed and a faster
 startup and hot reloads. It is only meant for local development, and the setup
 for it can be found inside the plugin's `dev/` directory.
+
+### Other Plugin Types
```

Comment:
We typically call these "plugin libraries" or "plugin library packages", they're not really plugins as such. This also comes across in the package role where they're all `web-library`, `common-library`, or `node-library`

Reply:
Sounds good to me, I've updated it 👍